### PR TITLE
Pp 71 create pet card

### PR DESCRIPTION
**Pull request by LiubovCass**
Created PetCard with React Bootstrap 5.2

### CODE REVIEW

**Review by AlonaVladymyrovaTrinity — APPROVED** [review]

File: src/component/Home/PetCard.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import whiteCat from '../../images/whiteCat.jpg';
```
Comment (badgerbadgers) [inline src/component/Home/PetCard.js:3]: Looks like the whiteCat.jpg image is missing and leads to an error when rendering the component.

**Comment by badgerbadgers** [comment]
Screenshot 2023-05-25 at 4 06 59 PM
Was able to render the component looks good.

**Review by badgerbadgers — CHANGES_REQUESTED** [review]
Will approve once the whiteCat image has been uploaded or removed.

**Review by badgerbadgers — APPROVED** [review]
Approving this since Oksana is aware of the whiteCat image missing.

**Review by OxiBo — APPROVED** [review]
Looks good. I added whiteCat image which is serving as a placeholder for now.

**Comment by AlonaVladymyrovaTrinity** [comment]
> Looks good. I added whiteCat image which is serving as a placeholder for now.

Thank you I really appreciate it!

**Comment by LiubovCass** [comment]
Thank you Oksana!

**Pull request merged**